### ISSUE TITLE
Is there a way to merge the learning histories of two different accounts?

### ISSUE THREAD

**Issue by ChihoHarada** [MicrosoftDocs/feedback#880]
Hello, 

I have several Microsoft accounts (personal, company's and trial) and went through several different learning path in Microsoft Docs using each of them. 
As a result, I have separated learning histories with each of my mail accounts.

Is there any way to merge all learning histories (because It's annoying sometimes...)

I'm new to GitHub and am not used to search existing issues. 
If there is a similar issue already, please forgive me.

Regards,

**Comment by Powerhelmsman** [comment]
@yunyunyunyun @dastaffo Can you take a look at this feedback?

**Comment by saldana** [comment]
Linking different accounts is in our roadmap within the next few months. Right now I do not have a date where we can deliver this functionality though.

**Comment by nosajnap** [comment]
@saldana  Hi, I was wondering if there was a progress update on this? Would it possibly be rolled out soon? Thanks!

**Comment by saldana** [comment]
@jialge can you provide an update to @nosajnap?

**Comment by rafferty** [comment]
+1 on this. More specifically, I'm hoping that I have been training using my work account, then I realize that none of my learning history is in my personal Microsoft account.

**Comment by saldana** [comment]
@jialge and @Sampy can you reply back to the customers?

**Comment by Sampy** [comment]
As Sandra said, this is on our road map to allow merging of two existing accounts and all of their learning history. Sooner on our road map is the ability to have two identities (for example a personal Microsoft Account and an organizational AAD account) linked to a single profile on Docs. This first feature will only allow adding identities that aren't already mapped to a profile, however. If you have a profile with little or no progress on it, you can delete it from your profile settings page to free up the identity for linking to a different profile.

Linking multiple identities is coming very soon so keep an eye out to the Docs blog and Twitter.

**Comment by canoas** [comment]
How soon can this be made available? 
PS: Now we have MS account managers monitoring MS Learn progress of partners and it is not possible to merge accounts yet. Older guys have mixed achievements in both old LiveID accounts and in newer Office365 accounts. 
 
Please share some progress on this.
Thank you

**Comment by dstarr** [comment]
Yes, please. I desperately need this ability OR to synch the learning from one account to another.

**Comment by saldana** [comment]
account linking is available already. Go to your profile and then settings, and then you will see the option for account linking.

![image](https://user-images.githubusercontent.com/3681607/76788894-058c3580-6779-11ea-8027-00727fff732c.png)

**Comment by JCR0mero** [comment]
And if the system says:
![image](https://user-images.githubusercontent.com/63334269/78748244-23d8f180-7931-11ea-8a68-f76d9c14ac2b.png)

**Comment by canoas** [comment]
> account linking is available already. Go to your profile and then settings, and then you will see the option for account linking.
> 
> ![image](https://user-images.githubusercontent.com/3681607/76788894-058c3580-6779-11ea-8027-00727fff732c.png)

no, it does not work when both logins already have existing profiles on mslearn.

**Comment by saldana** [comment]
You are right, @canoas. That is a shortcoming of this feature. Only one account can have a profile.  @yunyunyunyun or @dastaffo do you know if this is in profile's roadmap?

**Comment by rootsmusic** [comment]
#3140
#2726

**Comment by saldana** [comment]
@sharriexie I am assigning this to you. Can you please address this? Thanks!
